db/flush_job.cc

```diff
@@ -297,6 +303,11 @@ Status FlushJob::Run(LogsWithPrepTracker* prep_tracker,
            << (IOSTATS(cpu_read_nanos) - prev_cpu_read_nanos);
   }
 
+  // Clean up mempurge output memtables flushed to SST.
```

Comment:
flushed to SST? I am confused about what `purged_mems` holds

Reply:
My bad, the name `purged_mems` is visibly confusing. I've replaced it with "`full_output_mems`". It contains the memtables resulting from the mempurge that are filled up to capacity and need to be flushed to storage.